flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -129,11 +128,11 @@ public static NetworkEnvironment fromConfiguration(
 			maxJvmHeapMemory,
 			localTaskManagerCommunication,
 			taskManagerAddress);
-		return NetworkEnvironment.create(networkConfig, taskEventPublisher, metricGroup, ioManager);
+		return create(networkConfig, taskEventPublisher, metricGroup, ioManager);
 	}
 
 	public static NetworkEnvironment create(
-			NetworkEnvironmentConfiguration config,
+		NetworkEnvironmentConfiguration config,
```

Comment:
See the changes for commit `[FLINK-11392][network] Introduce ShuffleEnvironment interface`, this indentation should not be changed.